web/packages/teleterm/src/main.ts

```diff
@@ -142,25 +167,15 @@ async function initializeApp(): Promise<void> {
     app.exit();
   });
 
-  // On Windows/Linux: Re-launching the app while it's already running
-  // triggers 'second-instance' (because of app.requestSingleInstanceLock()).
-  //
-  // On macOS: Re-launching the app (from places like Finder, Spotlight, or Dock)
-  // does not trigger 'second-instance'. Instead, the system emits 'activate'.
-  // However, launching the app outside the desktop manager (e.g., from the command
-  // line) does trigger 'second-instance'.
-  app.on('second-instance', () => {
-    windowsManager.focusWindow();
-  });
-  app.on('activate', () => {
-    windowsManager.focusWindow();
+  app.on('web-contents-created', (_, webContents) => {
```

Comment:
It doesn't matter much, but I think this should be called before creating the window (in `app.whenReady`).